components/script/dom/dissimilaroriginwindow.rs

```diff
@@ -167,9 +167,9 @@ impl DissimilarOriginWindowMethods for DissimilarOriginWindow {
     }
 
     // https://html.spec.whatwg.org/multipage/#dom-opener
-    fn Opener(&self, _: JSContext) -> JSVal {
+    fn Opener(&self, _: JSContext, mut retval: MutableHandleValue) {
         // TODO: Implement x-origin opener
-        UndefinedValue()
+        retval.set(UndefinedValue());
```

Comment:
is `retval` not undefined by default?

Reply:
Not necessarily, that depends on the caller (although in this case the codegen). I think this is more correct and being more explicit is always better.